config/dev/aks-credentials.yaml

```diff
@@ -25,3 +25,13 @@ spec:
     kind: Secret
     name: azure-aks-credential
     namespace: ${NAMESPACE}
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
i do not quite understand the purpose of this object along with the similar in the `config/dev/aws-credentials.yaml`, could you elaborate on this?

Reply:
discussed personally